src/payment/strategies/klarna/klarna-payment-strategy.spec.ts

```diff
@@ -179,31 +282,19 @@ describe('KlarnaPaymentStrategy', () => {
 
             await strategy.initialize({ methodId: paymentMethod.id, klarna: { container: '#container' } });
 
-            strategy.execute(payload);
+            await strategy.execute(payload);
```

Comment:
if not required is better not to add await, these test seems to only be performing assertions on what happens when `strategy.execute(payload)` is executed, not necessarily making assertions of what happens when it finishes.